lib/cocoapods/command/repo/add.rb

```diff
@@ -36,7 +36,7 @@ def validate!
           end
           if @name == 'master' || @url =~ %r{github.com[:/]+cocoapods/specs}i
             raise Informative,
-                  'To setup the master specs repo, please run `pod setup`.'
+                  'Master repo via git has been replaced with a CDN-based solution.'
```

Comment:
Should we only `raise` if the name is `'master'`? That way those that still want to use the GIT repo (ex. maybe they have some tooling built around it) can do so by manually adding the URL

Reply:
also maybe we should just map `master` => `trunk`?

Reply:
Removed the git URL. 

My idea is to have a clean start for Trunk in new projects + self-contained migration commands.

Reply:
yup just want to make sure projects that manually include `source 'https://github.com/CocoaPods/Specs'` still work